wiki/Options/en.md

```diff
@@ -29,7 +29,7 @@ If you are a guest, these options will be shown:
 | `Remember Username` | If checked, the game client will remember the entered username. | Checkbox | `Enabled` |
 | `Remember Password` | If checked, the game client will remember the entered password. This will also keep you signed in to the game client. | Checkbox | `Disabled` |
 | `Sign In` | Sign in to osu! with the given credentials. Pressing `Enter` in the username or password forms also works. | Button |  |
-| `Create an account` | Redirects you to [Create Account](https://osu.ppy.sh/p/register) using your default web browser. | Button |  |
+| `Create an account` | Lets you create an account in-game. | Button |  |
```

Comment:
maybe `Opens account registration screen` is better?

Reply:
Yes.